docker/Dockerfile

```diff
@@ -70,7 +70,7 @@ ENV VERSION=${VERSION}
 ENV GIT_COMMIT=${GIT_COMMIT}
 
 # Install must-gather dependency: `kubectl`
-RUN OS_ARCH=${TARGETARCH/x86_64/amd64} && OS_ARCH=${OS_ARCH/aarch64/arm64} && curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/${OS_ARCH}/kubectl
+COPY ./kubectl ./kubectl
```

Comment:
revert this line

Reply:
done by [commit](https://github.com/easystack/gpu-operator/pull/1/commits/cebe69cd815540e0b082adfec9c77544a7616c4b)